### ISSUE TITLE
Use average value

### ISSUE THREAD

**Issue by ohobby** [sbamueller/FeinstaubMelder#12]
Standard is to use the maximum value of PM10 when checking.
It would also be good to use the Average. (possibly in addition to the maximum measurement) which then tweeted hourly. Possibly, also use and evaluate the PM2.5 value as a measured value.

**Comment by miskaknapek** [comment]
Right, that could certainly be interesting - to see/read the average. 
Currently, the link to the web based sensor data provides a timeline a month back, but eg someone on a mobile phone, it would be much easier to include the average in the tweet. 

However, the LuftDaten API doesn't currently support fetching historical data, so one would need to either set up an API that does, or store the sensor data locally somehow.

**Comment by ohobby** [comment]
The Luftdaten api, have also 5 minutes, 1 houre and 1 day average data. https://github.com/opendata-stuttgart/meta/wiki/APIs
